BD-3790 Apply template standards to Custom Data (#8667)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/custom_data/custom_events/nested_objects.md

## Step 1: Generate a schema

> To access the nested data in your custom event, generate a schema for each event with nested event properties. Generate a schema by following these steps:

Reviewer: ```suggestion
You can access the nested data in your custom event by generating a schema for each event with nested event properties. To generate a schema:
```

### Liquid templating

> The following Liquid templating examples demonstrate how to reference the nested properties saved from the preceding API request and use them in your Liquid messaging. Using Liquid and dot notation, the template will the nested data to find the specific node you want to include in your messages.

Reviewer: ```suggestion
The following shows how to create a Liquid template that references the nested properties requested from the [previous API request](#api-request-body).
```

Reviewer: **Note:** the current change (not my suggestion above) seems to have a type/missing word after "the template will...".

My suggestion helps trim down the content, but _may_ not accurately describe the original intent, so please re-read before committing.
